manifests/subnet.pp

```diff
@@ -0,0 +1,69 @@
+# Define a DHCP subnet
+# @param network
+#   The network range (without the mask) from where IP's will be served
+# @param mask
+#   The subnet mask for our network 
+# @param pools
+#   Specify a pool of addresses that will be treated differently than another pool of addresses,
+#   even on the same network segment or subnet.
+# @param gateway
+#   This option specifies a list of comma-separated IP addresses for routers on the client's subnet.
+#   Routers should be listed in order of preference.
+# @param options
+#   Custom DHCP option statements
+#   DHCP option statements always start with the option keyword, followed by an option name, followed by option
+#   data. The option names and data formats are described below. It is not necessary to exhaustively specify all
+#   DHCP options - only those options which are needed by clients
+# @param parameters
+#   Custom DHCP parameters
+#   Each element is added as a separate line
+# @param mtu
+#   This option specifies the MTU to use on this interface. The minimum legal value for the MTU is 68. 
+# @param nameservers
+#   Specifies a list of Domain Name System (STD 13, RFC 1035) name servers avail‐
+#   able to the client. Servers should be listed in order of preference.
+# @param pxeserver
+#   This is used to specify the host address of the server from which the initial boot file
+#   (specified in the filename statement) is to be loaded. Server-name should be a numeric IP address or a
+#   domain name. If no next-server statement applies to a given client, the address 0.0.0.0 is used.
+# @param pxefilename
+#   This can be used to specify the name of the initial boot file which is to be loaded by a
+#   client. The filename should be a filename recognizable to whatever file transfer protocol the client can be
+#   expected to use to load the file.
+# @param domain_name
+#   This option specifies the domain name that client should use when resolving hostnames via the Domain Name
+#   System.
+# @param static_routes
+#   This option specifies a list of static routes that the client should install in its routing cache. If mul‐
+#   tiple routes to the same destination are specified, they are listed in descending order of priority.
```

Comment:
I don't know how well puppet-strings understands splitting words like this. Probably safer to do this:
```suggestion
#   This option specifies a list of static routes that the client should install in its routing cache. If
#   multiple routes to the same destination are specified, they are listed in descending order of priority.
```

There's more cases in this file where this comment applies and I won't comment on all of them.